code/modules/reagents/reagent_containers/glass/bottle.dm

```diff
@@ -299,4 +299,120 @@
+	name = "Flu virion culture bottle"
+	desc = "A small bottle. Contains H13N1 flu virion culture in synthblood medium."
+	icon_state = "bottle3"
+	spawned_disease = /datum/disease/advance/flu
+
+/obj/item/weapon/reagent_containers/glass/bottle/epiglottis_virion
+	name = "Epiglottis virion culture bottle"
+	desc = "A small bottle. Contains Epiglottis virion culture in synthblood medium."
+	icon_state = "bottle3"
+	spawned_disease = /datum/disease/advance/voice_change
+
+/obj/item/weapon/reagent_containers/glass/bottle/liver_enhance_virion
+	name = "Liver enhancement virion culture bottle"
+	desc = "A small bottle. Contains liver enhancement virion culture in synthblood medium."
+	icon_state = "bottle3"
+	spawned_disease = /datum/disease/advance/heal
+
+/obj/item/weapon/reagent_containers/glass/bottle/hullucigen_virion
+	name = "Hullucigen virion culture bottle"
+	desc = "A small bottle. Contains hullucigen virion culture in synthblood medium."
+	icon_state = "bottle3"
+	spawned_disease = /datum/disease/advance/hullucigen
+
+/obj/item/weapon/reagent_containers/glass/bottle/pierrot_throat
+	name = "Pierrot's Throat culture bottle"
+	desc = "A small bottle. Contains H0NI<42 virion culture in synthblood medium."
+	icon_state = "bottle3"
+	spawned_disease = /datum/disease/pierrot_throat
+
+/obj/item/weapon/reagent_containers/glass/bottle/cold
+	name = "Rhinovirus culture bottle"
+	desc = "A small bottle. Contains XY-rhinovirus culture in synthblood medium."
+	icon_state = "bottle3"
+	spawned_disease = /datum/disease/advance/cold
+
+/obj/item/weapon/reagent_containers/glass/bottle/retrovirus
+	name = "Retrovirus culture bottle"
+	desc = "A small bottle. Contains a retrovirus culture in a synthblood medium."
+	icon_state = "bottle3"
+	spawned_disease = /datum/disease/dna_retrovirus
+
+/obj/item/weapon/reagent_containers/glass/bottle/gbs
+	name = "GBS culture bottle"
+	desc = "A small bottle. Contains Gravitokinetic Bipotential SADS+ culture in synthblood medium."//Or simply - General BullShit
+	icon_state = "bottle3"
+	amount_per_transfer_from_this = 5
+	spawned_disease = /datum/disease/gbs
+
+/obj/item/weapon/reagent_containers/glass/bottle/fake_gbs
+	name = "GBS culture bottle"
+	desc = "A small bottle. Contains Gravitokinetic Bipotential SADS- culture in synthblood medium."//Or simply - General BullShit
+	icon_state = "bottle3"
+	spawned_disease = /datum/disease/fake_gbs
+
+/obj/item/weapon/reagent_containers/glass/bottle/brainrot
+	name = "Brainrot culture bottle"
+	desc = "A small bottle. Contains Cryptococcus Cosmosis culture in synthblood medium."
+	icon_state = "bottle3"
+	spawned_disease = /datum/disease/brainrot
+
+/obj/item/weapon/reagent_containers/glass/bottle/magnitis
+	name = "Magnitis culture bottle"
+	desc = "A small bottle. Contains a small dosage of Fukkos Miracos."
+	icon_state = "bottle3"
+	spawned_disease = /datum/disease/magnitis
+
+/obj/item/weapon/reagent_containers/glass/bottle/wizarditis

```

Comment:
@Fox-McCloud Wizarditis is in the contraband. Wizard would have to hack the machine